### ISSUE TITLE
model-best

### ISSUE THREAD

**Issue by Ma-PC** [markub3327/HAR-Transformer#6]
I am very interested in your paper.
How to save your own model weights through training？      model.load_weights("./save/model-best.h5")
looking forward to your reply.

**Comment by markub3327** [comment]
Hi @Ma-PC,

   the model is by default stored in the WanDB Project. If you want to do it manually please add a line after `model.fit()`:
`model.save_weights("./save/model-best.h5")` for H5 format or `model.save_weights("./save/model-best")` for TF format.

If you want only the best model weights, add the flag: `restore_best_weights=False` to the `EarlyStopping()` callback.

Thanks.

**Comment by Ma-PC** [comment]
thank you for your reply。
I found weight.h5 from running. Is this the best model to automatically save?
![1](https://github.com/markub3327/HAR-Transformer/assets/132666100/2da49927-ce0a-41c8-94d8-4881c403daab)
![2](https://github.com/markub3327/HAR-Transformer/assets/132666100/3ce91f00-9e19-4877-8782-f3f1d81934a4)

**Comment by markub3327** [comment]
Yes, @Ma-PC . The screenshot depicts the best model file created by `WandbCallback()` on maximal validation accuracy.

Thanks.

**Comment by markub3327** [comment]
@Ma-PC,

  It's all from your side. Can I help you with any further questions? Can I close this issue?

Thanks.

**Comment by Ma-PC** [comment]
I've closed the issue, I managed to run the code successfully using my own dataset. Thank you for your prompt responses, I appreciate it.
